README.md

```diff
@@ -1 +1,78 @@
+* [Java 21](https://www.oracle.com/uk/java/technologies/downloads/#java21)
+* [Maven](https://maven.apache.org/download.cgi)
+* [MongoDB](https://www.mongodb.com/) 
+* [Apache Kafka](https://kafka.apache.org/)
+* Internal Companies House core services
+
+You will also need a REST client (e.g. Postman or cURL) if you wish to interact with the `psc-verification-api` service endpoints.
+
+## Building and Running Locally
+
+1. From the command line, in the same folder as the Makefile run `make clean build`
+1. Configure project environment variables where necessary (see below).
+1. Ensure dependent Companies House services are running within the Companies House developer environment
+1. Start the service in the CHS developer environment
+1. Send a GET request using your REST client to `/psc-verification-api/healthcheck`. The response should be `200 OK` with `status=UP`.
+1. A MongoDB instance named _TBC_ and the following collections are required: _TBC_
+
+
+
+Configuration
+-------------
+System properties for the `psc-verification-api` are defined in `application.properties`. These are normally configured per environment.
+
+Variable| Description                                                                           | Example                                    |
+-------------------|---------------------------------------------------------------------------------------|--------------------------------------------|
+_TBC_| The name of the collection responsible for storing PSC verification filings           | verifications                              |
+MONGODB_URL| The URL of the MongoDB instance where documents and application data should be stored | mongodb://mongohost:27017/verifications    |
+ PLANNED_MAINTENANCE_START_TIME               | Datetime for start of out-of-service period                                           | 14 July 24 00:30 +01 |optional; requires End time (see below)
+ PLANNED_MAINTENANCE_END_TIME                 | Datetime for end of out-of-service period                                             | 2 Dec 23 02:30 GMT|optional; requires Start time (see above)
+ PLANNED_MAINTENANCE_MESSAGE                  | Message output during the out-of-service period                                       | Service is undergoing planned maintenance  |optional; default value *UNAVAILABLE - PLANNED MAINTENANCE*
+
+### Notes
+
+Planned maintenance format: `d MMM yy HH:mm z|x` where
+- `MMM` is the 3-letter month abbrev. (case sensitive: e.g. `Nov` not `NOV`)
+- `z` is the zone short name e.g. `GMT`
+- `x` is the 2-digit zone offset from UTC e.g. `+01`  (= British Summer Time)
+
+> **CAUTION**: Use zone offset *+01* for Daylight Saving Time (British Summer Time). Zone short name *BST* denotes Bangladesh Standard Time (UTC+06) not British Summer Time (UTC+01).
+
+
+## Building the docker image
+
+    mvn -s settings.xml compile jib:dockerBuild -Dimage=169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/psc-verification-api
+
```

Comment:
I think this needs to be `416670754337.dkr.ecr.eu-west-2.amazonaws.com/psc-verification-api:latest`